larpandoracontent/LArHelpers/LArGeometryHelper.cc

```diff
@@ -331,31 +331,64 @@ CartesianVector LArGeometryHelper::ProjectDirection(const Pandora &pandora, cons
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
-float LArGeometryHelper::GetWireZPitch(const Pandora &pandora)
+float LArGeometryHelper::GetWireZPitch(const Pandora &pandora, const float maxWirePitchWDiscrepancy)
```

Comment:
Should just make this function call the second function with specified view TPC_VIEW_W. This kind of function is then typically inline in Pandora.

Note that this is the kind of operation where we get into trouble when w no longer necessarily is equivalent to z, as Andy mentioned in his e-mail yesterday.